app/full_text_search.py

```diff
@@ -158,6 +213,7 @@ def create_jp_location_fields(**kwargs):
                 )
     return fields
 
+
 def create_document(record_id, repo, **kwargs):
```

Comment:
As discussed, please refactor the code to create a function similar to create_jp_name_fields which supports both romanization.